apps/meteor/client/components/avatar/DirectRoomAvatar.tsx

```diff
@@ -0,0 +1,18 @@
+import { useUserId } from '@rocket.chat/ui-contexts';
+import React, { memo, ReactElement, ComponentProps, useMemo } from 'react';
+
+import { useEndpointData } from '../../hooks/useEndpointData';
+import RoomAvatar from './RoomAvatar';
+
+const DirectRoomAvatar = ({ room, ...rest }: ComponentProps<typeof RoomAvatar>): ReactElement => {
+	const ownId = useUserId();
+	const otherId = room.uids?.find((uid) => uid !== ownId);
+	const { value: data } = useEndpointData(
```

Comment:
I think we are better of using the usePresence hook. Even though it only brings offline users, getting the info from the endpoint won't be reactive, and we'll have the same problem again.